src/Microsoft.Health.Dicom.Core/Features/Update/UpdateInstanceOperationService.cs

```diff
@@ -88,6 +100,9 @@ public class UpdateInstanceOperationService : IUpdateInstanceOperationService
         try
         {
             var operation = await _client.StartUpdateOperationAsync(operationId, updateSpecification, partitionKey, cancellationToken);
+
+            string input = JsonSerializer.Serialize(updateSpecification, _jsonSerializerOptions.Value);
+            _telemetryClient.ForwardLogTrace("Dicom update operation", operationId.ToString(), input);
```

Comment:
As you mentioned earlier, how are we going handle if the payload size larger than 32KB?